apps/meteor/app/federation-v2/server/index.ts

```diff
@@ -71,6 +71,17 @@ const federationMessageServiceSender = FederationFactory.buildMessageServiceSend
 
 let cancelSettingsObserver: () => void;
 
+const onFederationEnabledStatusChanged = async (isFederationEnabled: boolean): Promise<void> => {
+	federationBridge.onFederationAvailabilityChanged(isFederationEnabled);
+	if (isFederationEnabled) {
+		federationBridge.logFederationStartupInfo('Running Federation V2');
+		FederationFactory.setupActions(federationRoomServiceSender, federationMessageServiceSender);
+		require('./infrastructure/rocket-chat/slash-commands');
```

Comment:
Replace this and the other requires with an await import - some of our tools don't deal well with requires.

```suggestion
		await import('./infrastructure/rocket-chat/slash-commands');
```